edk2toollib/uefi/edk2/path_utilities.py

```diff
@@ -19,64 +19,91 @@
 
 
 class Edk2Path(object):
+    """Represents edk2 file paths.
 
-    def __init__(self, ws: os.PathLike, packagepathlist: Iterable[os.PathLike], error_on_invalid_pp: bool = True):
-        """ An Edk2Path object is an object that can be used to resolve edk2 relative paths
+    Can be used to resolve relative paths.
+    """
+
+    def __init__(self, ws: os.PathLike, package_path_list: Iterable[os.PathLike], error_on_invalid_pp: bool = True):
+        """Constructor.
 
         Args:
-            ws: absolute path or cwd relative path of the workspace.
-            packagespathlist: list of packages path.
+            ws (os.PathLike): absolute path or cwd relative path of the workspace.
+            packagespathlist (Iterable[os.PathLike]): list of packages path.
```

Comment:
this arg (packagespathlist) is wrong since the arg name was changed to use snake case

Reply:
@makubacki no need to fix this. I'm currently working on a pydocstyle for pytool-library and will fix it. You'll have conflicts to work through due to my changes to all documentation anyway, so you'll end up doing fixing it twice.

Reply:
Also, pydocstyle catches issues like this, which is nice :)

Reply:
It's a very simple change, no need for it to be wrong here. 

Updated in PR update https://github.com/tianocore/edk2-pytool-library/pull/186/commits/e21c87394eb5f15e4679f64235f30f15ac1280ca